lib/hu_board/account_helpers.rb

```diff
@@ -25,7 +25,7 @@ def subscription_active?(customer)
       cus = customer && customer[:stripe][:customer]
       if cus && cus[:subscriptions][:total_count] > 0
         sub = cus[:subscriptions][:data][0]
-        return sub[:status] == "active" || sub[:status] == "trialing"
+        return sub[:status] == "active" || sub[:status] == "trialing" || sub[:status] == 'past_due'
```

Comment:
Should we use the `%{ trialing past_due }.include?(plan[:status])` syntax here, too?